models/comments.models.js

```diff
@@ -25,3 +26,14 @@ exports.insertComment = async (article_id, username, body) => {
 
   return result.rows[0];
 };
+
+// REMOVE
+
+exports.removeCommentById = async (comment_id) => {
+  const result = await db.query(
+    "DELETE FROM comments WHERE comment_id = $1 RETURNING *;",
```

Comment:
Would there be anything to return from this query if something has been deleted?

Reply:
The returning does return the deleted comment, though I'm not using it for anything at the moment.